pkg/curl/gen.lua

```diff
@@ -21,21 +21,24 @@ build('cat', '$outdir/curl_config.h', {
 	'$builddir/probe/SIZEOF_TIME_T',
 })
 
+build('touch', '$outdir/include/curl/websockets.h')
+
```

Comment:
A bunch of files include this despite it being disabled. I did a build through of core, net, and media and nothing fails if this is just an empty header.

Reply:
I guess I should have checked but its `curl/curl.h` thats including it. I guess the actual header could just be installed since the entirety of `lib/ws.c` besides the headers is enclosed in `#ifdef USE_WEBSOCKETS`.

Reply:
I think just installing the header is the best solution.